src/Libraries/CoreNodeModels/CoreNodeModelsImages.resx

```diff
@@ -768,6 +768,25 @@
         L0B4qfoOUL6IiIiIiIiIiIiIiIiIiIiIiIiIyG73B7BuavcidkHUAAAAAElFTkSuQmCC
 </value>
   </data>
+  <data name="CoreNodeModels.Logic.RefactoredIf.Large" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
```

Comment:
I don't think you need to do this (i.e. add new entries here). If I'm not wrong, you can simply change the name of the existing `If` node class bitmap to the new class and rename the png files too.

Reply:
Yeah sounds good, forgot that we won't be showing the old node in the library anymore.